definitions/ext-nas/readynas-dashboard.json

```diff
@@ -205,7 +205,7 @@
 						"nrqlQueries": [
 							{
 								"accountId": 0,
-								"query": "FROM Metric SELECT latest(kentik.snmp.volumeRAIDLevel) AS 'RAID', latest(kentik.snmp.volumeSize)/1024 AS 'Total GB', latest(kentik.snmp.volumeFreeSpace)/1024 AS 'Free GB', (latest(kentik.snmp.volumeSize) - latest(kentik.snmp.volumeFreeSpace)) / latest(kentik.snmp.volumeSize) *100 AS '% Utilization' FACET volumeName WHERE provider = 'kentik-nas' AND SysObjectID = '.1.3.6.1.4.1.4526.100.16'"
+								"query": "FROM Metric SELECT latest(kentik.snmp.volumeRAIDLevel) AS 'RAID', latest(kentik.snmp.volumeSize)/1024 AS 'Total GB', latest(kentik.snmp.volumeFreeSpace)/1024 AS 'Free GB' (latest(kentik.snmp.volumeSize) - latest(kentik.snmp.volumeFreeSpace)) / latest(kentik.snmp.volumeSize) *100 AS '% Utilization' FACET volumeName WHERE provider = 'kentik-nas' AND SysObjectID = '.1.3.6.1.4.1.4526.100.16'"
```

Comment:
Same here, these queries with the comma removal will fail, were these changes intended?

Reply:
Same as above, must have changed when I ran sanitize dashboards